### ISSUE TITLE
Seasickness

### ISSUE THREAD

**Issue by jrosindell** [OneZoom/OZtree#931]
Feedback 

"lots of students got motion sick from the repeated flys, I made most switch to jump to, But that's quite disorientating. I feel like the acceleration is to linear in increase and caps out too soon, I think an option for a slightly faster zoom rather than no zoom would be good. Or using a bezier curve acceleration that speeds up in the middle."

This is by no means a unique comment - lots of people have previously comments on the same thing.  I think that on the default zoom mode that speeds up a lot in the middle, and maybe pauses for longer on the common ancestor so that the total duration of zoom is much less but the orientation is clearer. 

Not much mention of seasickness in other issues but something in #91 seems vaguely relevant.

**Comment by jrosindell** [comment]
#525 is also relevant here

**Comment by davidebbo** [comment]
#920 is also related. I do personally use jump mode, more to save time than because of seasickness.

**Comment by lentinj** [comment]
I was talking about this with a friend at the weekend, co-incidentally. I'm slightly nervous that [the smooth flights](https://github.com/OneZoom/OZtree/pull/925) will make this worse, and had been wondering if a low-animation mode with ~2 fps would help. But it'd be good to find some guidance over what triggers motion-sickess first.

**Comment by jaredkhan** [comment]
Yeah I've had friends say it makes them feel sick too. 

Personally I find the smoother flight and higher res makes it easier to follow the motion. It could be worth testing that branch with a complainant to see if they find it better or worse.

**Comment by lentinj** [comment]
Yeah agreed, it looks a lot more like an actual flight now. This is partially what made me think of it.

I've a vague recollection that motion at the extremities of vision that is particularly triggering, but I can't find anything to back me up (this was why I was trying to pump my friend for clues).

The tendency for the visualisations (bar fern) to spiral I don't think is doing us any favours here. If it came up again it might be interesting to see if the user thinks fern (or any other visualisation) is less triggering.
